pgcli/main.py

```diff
@@ -103,16 +103,13 @@ def cli(database, user, password, host, port):
             try:
                 _logger.debug('sql: %r', document.text)
                 res = pgexecute.run(document.text)
-                output = []
```

Comment:
The old code here seems buggy and confusing. Currently the list returned by pgexecute.run always contains only 1 element so why is the for loop even needed?